### DISCUSSION TITLE
doc-site Components documentation - Change items

### DISCUSSION THREAD

**Discussion by gargajit** [fastn-community/doc-site#7] [General]
- [x] index

  - [x] Components content: 
```
Components are like Lego blocks for building websites. They are pre-designed 
building blocks that serve specific purposes or display specific content, 
allowing you to easily create a complete webpage by combining and customizing 
them.
```

- [x] General:

  - [x] Do we really need this line in all the document: `In this section, we will learn more about -- ds.code component and its usage.`
  - [x] UI must be improved

**Comment by gsalunke** [comment]
> * [x]  index
>   
>   * [x]  Components content:
> 
> ```
> Components are like Lego blocks for building websites. They are pre-designed 
> building blocks that serve specific purposes or display specific content, 
> allowing you to easily create a complete webpage by combining and customizing 
> them.
> ```
> 
> * [x]  General:
>   
>   * [x]  Do we really need this line in all the document: `In this section, we will learn more about -- ds.code component and its usage.`
>   * [x]  UI must be improved

* [x] Add code block of each component use case
* [x] [Click here](http://127.0.0.1:8000/blog/) to see it in -- Change with: To see it in action - [Click here](http://127.0.0.1:8000/blog/) 
* [x]  Bring right-sidebar with Index on all pages.
* [x] Changing Color Scheme - change content like - Right Sidebar, as its in old format
* [x] Changing Typography - change content like - Right Sidebar, as its in old format
